packages/@aws-cdk/aws-stepfunctions-tasks/lib/databrew/start-job-run.ts

```diff
@@ -0,0 +1,80 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for starting a job run with StartJobRun
+ * @experimental
+ */
+export interface DataBrewStartJobRunProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * DataBrew Job to run
+   */
+  readonly name: string;
+}
+
+/**
+ * Start a Job run as a Task
+ *
+ * @see https://docs.aws.amazon.com/step-functions/latest/dg/connect-databrew.html
+ * @experimental
+ */
+export class DataBrewStartJobRun extends sfn.TaskStateBase {
+
+  private static readonly SUPPORTED_INTEGRATION_PATTERNS: sfn.IntegrationPattern[] = [
+    sfn.IntegrationPattern.REQUEST_RESPONSE,
+    sfn.IntegrationPattern.RUN_JOB,
+  ];
+
+  protected readonly taskMetrics?: sfn.TaskMetricsConfig;
+  protected readonly taskPolicies?: iam.PolicyStatement[];
+
+  private readonly integrationPattern: sfn.IntegrationPattern;
+
+  /**
+   * @experimental
+   */
+  constructor(scope: Construct, id: string, private readonly props: DataBrewStartJobRunProps) {
+    super(scope, id, props);
+    this.integrationPattern = props.integrationPattern ?? sfn.IntegrationPattern.REQUEST_RESPONSE;
+
+    validatePatternSupported(this.integrationPattern, DataBrewStartJobRun.SUPPORTED_INTEGRATION_PATTERNS);
+
+    this.taskPolicies = [
+      new iam.PolicyStatement({
+        resources: [
+          cdk.Stack.of(this).formatArn({
+            service: 'databrew',
+            resource: 'job',
+            // If the name comes from input, we cannot target the policy to a particular ARN prefix reliably.
+            resourceName: sfn.JsonPath.isEncodedJsonPath(this.props.name) ? '*' : this.props.name,
```

Comment:
does this mean if the name comes from the input, this state machine will now have permissions to call "any" glue data brew job within that account?

Reply:
Correct.

Reply:
got it, this is the best we can do when the name is supplied dynamically through state input

Reply:
If the resource for a Task is specified from the state input with JSONPath, it is only known at runtime so `"Resource": "*"` is required. The Step Functions console uses this approach to creating IAM roles based on the state machine definition.

Reply:
https://docs.aws.amazon.com/step-functions/latest/dg/service-integration-iam-templates.html